e2e/cypress/integration/dashboard-transloadit.spec.ts

```diff
@@ -15,4 +15,28 @@ describe('Dashboard with Transloadit', () => {
 
     cy.get('.uppy-StatusBar-statusPrimary').should('contain', 'Complete')
   })
+
+  it('should close assembly polling when cancelled', () => {
+    cy.get('@file-input').attachFile(['images/cat.jpg', 'images/traffic.jpg'])
+    cy.get('.uppy-StatusBar-actionBtn--upload').click()
+
+    cy.intercept({
+      method: 'GET',
+      url: '/assemblies/*',
+    }).as('assemblyPolling')
+    cy.intercept(
+      { method: 'PATCH', pathname: '/files/*', times: 1 },
+      { statusCode: 204, body: {} },
+    )
+    cy.intercept(
+      { method: 'DELETE', pathname: '/resumable/files/*', times: 1 },
+      { statusCode: 204, body: {} },
+    )
+    cy.wait('@assemblyPolling')
+    cy.get('.uppy-u-reset.uppy-StatusBar-actionCircleBtn[title="Cancel"]').click()
```

Comment:
Perhaps it might be nice to add data attributes to the most common things like the cancel and upload button

https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
Yep, especially since this could break due to i18n.